src/components/workMenuEntry.svelte

```diff
@@ -84,6 +90,7 @@
       h[highlightI].tagNames.splice(tagI, 1);
       return h;
     });
+    saveResumeDataToLocalStorage();
```

Comment:
Same here, I don't like having to explicitly call a function to save the data to local storage, when I could just do that automatically from within the corresponding functions. This may be tougher to fix since the way I wrote data handling back then was wonky.